### ISSUE TITLE
Ninja teleports can cause telegibbing to occur.

### ISSUE THREAD

**Issue by Kaedwuff** [Aurorastation/Aurora.3#6339]
![3362d661c16526d544b08e0e253dbdfc](https://user-images.githubusercontent.com/12869074/56472101-11b02b80-6420-11e9-8579-58f3b6a69248.png)

I'm not really sure what caused this to happen, but the ninja was rapidly teleport-dragging someone around and they got telegibbed on this wall.  If I had to take a guess, it's because while the ninja might have checks made on teleporting into solid objects, the person they are grabbing is still on another square next to them and there is no such check for them.  

This could end up being abused if people figure out how to successfully cause it to happen on purpose.

**Comment by Sindorman** [comment]
iirc it accounts for things you drag with you in teleportation. Right @LordFowl ?

**Comment by SonicGotNuked** [comment]
Yeah, I was the RD in that picture, and I can confirm it indeed telegibbed me with the ninja teleporting me around. I think I may have been in an aggressive grab at the time because I think kill-grab (red) puts them on the same turf.

**Comment by Sindorman** [comment]
In the screenshot you did tho teleported into tile that had dense object - hanger. Dense object cause you to loose limbs.

**Comment by SonicGotNuked** [comment]
Makes sense, but it is probably really easy to replicate and abuse. I think I already got a decent idea how.
